### PR TITLE
1.28.1 Release Notes

### PR DESCRIPTION
**Pull request by lzdanski**
Related to https://github.com/astronomer/updates.astronomer.io/pull/231

### CODE REVIEW

**Review by TJaniF — APPROVED** [review]
LGTM :)

File: astro/cli/release-notes.md [hunk 1]
```diff
@@ -18,6 +18,14 @@ This document provides a summary of all changes made to the [Astro CLI](cli/over
 
 - **Stable versions**: {{CLI_VER_LATEST}}, {{CLI_VER_2}}, and {{CLI_VER_3}}. See [Astro CLI release and lifecycle policy](cli/release-lifecycle-policy.md) for more information about support for CLI versions.
 
+## Astro CLI 1.28.1
+
+Release date: July 25, 2024
+
+## Bug fixes
+
+- Fixed a bug where Git ignored all hidden paths, causing Astro to not respect the `.airflowignore` file.
```
Comment (jeremybeard) [inline astro/cli/release-notes.md:27]: It was the CLI ignoring the files when bundling the DAGs.

**Review by jeremybeard — APPROVED** [review]
Approving ahead of commment.

File: astro/cli/release-notes.md (hunk 1 above)
Comment (lzdanski) [inline astro/cli/release-notes.md:27]:
```suggestion
- Fixed a bug where the CLI ignored files when bundling DAGs for code deploys.
```

**Pull request merged**